projects/angular/src/forms/combobox/_combobox.clarity.scss

```diff
@@ -190,129 +101,55 @@
-        clr-combobox-pill-margin,
-        variables.$clr_baselineRem_0_125 variables.$clr_baselineRem_0_25 0 0,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        background-color,
-        clr-combobox-pill-background-color,
-        combobox-variables.$clr-combobox-pill-background-color,
-        variables.$clr-use-custom-properties
-      );
-      border-width: variables.$clr_baselineRem_1px;
+      @include mixins.generate-typography-token(
+        'CAPTION-LG-11',
+        (
+          font-weight: combobox-variables.$clr-combobox-pill-font-weight,
+          font-size: combobox-variables.$clr-combobox-pill-font-size,
+          line-height: combobox-variables.$clr-combobox-pill-line-height,
+          letter-spacing: combobox-variables.$clr-combobox-pill-letter-spacing,
+        )
+      );
+      height: combobox-variables.$clr-combobox-pill-height;
+      margin: combobox-variables.$clr-combobox-pill-margin;
+      background-color: combobox-variables.$clr-combobox-pill-background-color;
+      border-width: tokens.$cds-global-space-1;
       border-style: solid;
+      align-items: center;
 
-      // ATM Remove fallback and property
-      @include mixins.css-var(
-        align-items,
-        clr-combobox-pill-align-items,
-        baseline,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        border-radius,
-        clr-combobox-pill-border-radius,
-        combobox-variables.$clr-combobox-pill-border-radius,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        border-color,
-        clr-combobox-pill-border-color,
-        combobox-variables.$clr-combobox-pill-border-color,
-        variables.$clr-use-custom-properties
-      );
+      border-radius: combobox-variables.$clr-combobox-pill-border-radius;
+      border-color: combobox-variables.$clr-combobox-pill-border-color;
 
-      @include mixins.css-var(
-        padding,
-        clr-combobox-pill-padding,
-        0 variables.$clr_baselineRem_2px 0 variables.$clr_baselineRem_4px,
-        variables.$clr-use-custom-properties
-      );
+      padding: combobox-variables.$clr-combobox-pill-padding;
 
       .clr-combobox-pill-content {
-        @include mixins.css-var(
-          color,
-          clr-combobox-pill-font-color,
-          combobox-variables.$clr-combobox-pill-font-color,
-          variables.$clr-use-custom-properties
-        );
-        font-size: variables.$clr-typography-xxsmalltext;
-        font-weight: variables.$clr-font-weight-regular;
-        @include mixins.css-var(
-          padding,
-          clr-combobox-pill-content-padding,
-          0 variables.$clr_baselineRem_8px 0 variables.$clr_baselineRem_4px,
-          variables.$clr-use-custom-properties
-        );
+        font-size: tokens.$cds-alias-typography-caption-font-size;
+        font-weight: tokens.$cds-alias-typography-caption-font-weight;
```

Comment:
Why do we need `font-size` and `font-weight` overrides? They are already set in `.clr-combobox-pill` rule.
```suggestion
```

Reply:
They indirectly affect the vertical size of the combobox container. At least the font-size one does.
It can be handled at the layout refactor stage.